evap/staff/tests/test_views.py

```diff
@@ -999,21 +999,59 @@ class TestGradeReminderView(WebTestStaffMode):
     def setUpTestData(cls):
         cls.manager = make_manager()
         cls.responsible = baker.make(UserProfile, first_name_given="Bastius", last_name="Quid")
-        cls.evaluation = baker.make(
+        cls.semester = baker.make(Semester)
+        cls.url = f"/staff/semester/{cls.semester.pk}/grade_reminder"
+
+        course_args = {"responsibles": [cls.responsible], "gets_no_grade_documents": False, "semester": cls.semester}
+        cls.course1 = baker.make(Course, name_en="A-Course1", name_de="Z-Course1", **course_args)
+        cls.course2 = baker.make(Course, name_en="Z-Course2", name_de="A-Course2", **course_args)
+
+        baker.make(
             Evaluation,
-            course__name_en="How to make a sandwich",
-            course__responsibles=[cls.responsible],
-            course__gets_no_grade_documents=False,
+            course=cls.course1,
             state=Evaluation.State.EVALUATED,
             wait_for_grade_upload_before_publishing=True,
+            _fill_optional=["name_de", "name_en"],
+            _quantity=2,
+        )
+        cls.course2_evaluation = baker.make(
+            Evaluation,
+            course=cls.course2,
+            state=Evaluation.State.IN_EVALUATION,  # blocking
+            wait_for_grade_upload_before_publishing=True,  # blocking
```

Comment:
On first read, I am not sure what is being blocked here, so maybe these comments should be rather placed in the tests that needs these values to be set?

Reply:
I removed the comments here.